x-pack/plugins/aiops/public/embeddable/embeddable_change_point_chart_factory.ts

```diff
@@ -0,0 +1,100 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import {
+  EmbeddableFactoryDefinition,
+  ErrorEmbeddable,
+  IContainer,
+} from '@kbn/embeddable-plugin/public';
+import { i18n } from '@kbn/i18n';
+import { type DataPublicPluginStart } from '@kbn/data-plugin/public';
+import { StartServicesAccessor } from '@kbn/core-lifecycle-browser';
+import { EMBEDDABLE_CHANGE_POINT_CHART_TYPE } from '../../common/constants';
+import type { AiopsPluginStart, AiopsPluginStartDeps } from '../types';
+import {
+  EmbeddableChangePointChart,
+  EmbeddableChangePointChartInput,
+} from './embeddable_change_point_chart';
+
+export interface EmbeddableChangePointChartStartServices {
+  data: DataPublicPluginStart;
+}
+
+export type EmbeddableChangePointChartType = typeof EMBEDDABLE_CHANGE_POINT_CHART_TYPE;
+
+export class EmbeddableChangePointChartFactory implements EmbeddableFactoryDefinition {
+  public readonly type = EMBEDDABLE_CHANGE_POINT_CHART_TYPE;
+
+  public readonly grouping = [
+    {
+      id: 'ml',
+      getDisplayName: () =>
+        i18n.translate('xpack.aiops.navMenu.mlAppNameText', {
+          defaultMessage: 'Machine Learning',
+        }),
+      getIconType: () => 'machineLearningApp',
+    },
+  ];
+
+  constructor(
+    private readonly getStartServices: StartServicesAccessor<AiopsPluginStartDeps, AiopsPluginStart>
+  ) {}
+
+  /**
+   * TODO
+   */
+  public isEditable = async () => {
+    return false;
```

Comment:
Unresolved todo

Reply:
I started working on the embeddable [initializer](https://github.com/elastic/kibana/blob/e4303a05597388d2d9cb5059725a12daeab43458/x-pack/plugins/aiops/public/embeddable/change_point_chart_initializer.tsx), but decided to finish it in a follow-up PR. There is a dedicated task in the issue (Support for adding/editing change point charts embeddable from the Dashboard app) for that.

Reply:
Follow up sounds great 👍